03-Bash/fix_owner/fix_owner.sh

```diff
@@ -54,13 +55,46 @@ get_file_grup(){
 	echo $(ls -nd "$1" | cut --delimiter=" " -f 4)
```

Comment:
Just for note. Another way for get file owner and group:
stat -c "%U %G" /path/to/file